### PR TITLE
Download 37363

### PR DESCRIPTION
**Pull request by steveoh**
[Instructions](https://github.com/agrc/wri-web/blob/43e54afaf5e63c6d9e1f64df96759b17d5582aff/scripts/download/Readme.md)
`WRI/Toolbox`
`Asynchronous`

### CODE REVIEW

File: src/app/mapControls/Download.js [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+    'dojo/dom-class',
+    'dojo/dom-construct',
+    'dojo/text!app/mapControls/templates/Download.html',
+    'dojo/topic',
+    'dojo/_base/declare',
+    'dojo/_base/lang',
+
+    'esri/tasks/Geoprocessor',
+
+    'xstyle/css!app/mapControls/resources/Download.css'
```
Comment (stdavis) [inline src/app/mapControls/Download.js]: This line should be removed. Should fix the failing build.

File: src/app/mapControls/Download.js [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+        //async gp task job id
+        jobId: null,
+
+        // parameters passed in via the constructor
+
+        // parentWidget: a reference to the parent widget
+        parentWidget: null,
+
+        // url: url to gp tool
+        url: null,
```
Comment (stdavis) [inline src/app/mapControls/Download.js]: This property doesn't look like it's used.

**Comment by steveoh** [comment]
both line comments should be addressed

**Comment by steveoh** [comment]
this is on hold for a minute @stdavis. It's going to become a toolbar item after a discussion with aaron.

**Comment by stdavis** [comment]
OK. Would be helpful to add a `secrets.py.sample` file at some point.

**Pull request closed without merging**